app/policies/concerns/policy_helpers.rb

```diff
@@ -63,12 +63,12 @@ def technical?
   end
 
   # Indicates whether the current user is member of the given organization
-  def member_of?(organization)
+  def member_of? organization
```

Comment:
My prefered IDE does not like definitions of methods without parenthesis and the help will refer to this: https://github.com/bbatsov/ruby-style-guide#syntax